sdk/spring/spring-cloud-azure-docker-compose/pom.xml

```diff
@@ -63,6 +63,12 @@
       <groupId>com.azure</groupId>
       <artifactId>azure-storage-blob</artifactId>
       <version>12.29.0</version> <!-- {x-version-update;com.azure:azure-storage-blob;dependency} -->
+      <exclusions>
+        <exclusion>
+          <groupId>com.fasterxml.jackson.core</groupId>
+          <artifactId>jackson-core</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
Why do we need to exclude `jackson-core` here?

Reply:
It introduces conflicting dependencies, which can cause `NoSuchMethodError: ParserMinimalBase` error